src/components/shared/Dialog/dialogTypes/ConfirmLogout.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
```

Comment:
Why is the file name ConfirmLogout when we are trying to make it generic? Very small mistakes are being pointed out in every review and PRs. Please try to learn from mistakes and it would help us save time and move the project move faster.

Reply:
@akshatnitd sorry for the inconvenience, but a file named `ConfirmDialog` is already present

Reply:
@Dilshaad21 Why is this file even needed in that case?

Reply:
I thought a new Modal should be used other than `confirmDialog` as it's generally used for confirmations, not Logout.

Reply:
@Dilshaad21 In that case, Cancel can be added to the existing modal. Doesn't make a difference.

Reply:
@akshatnitd Ok then I'll add cancel to the `confirmDialog` modal and make changes changes accordingly